frontend/playground-frontend/src/components/Home/TrainButton.js

```diff
@@ -46,23 +46,34 @@ const TrainButton = (props) => {
       const obj_list_item = obj_list[i];
       const parameters = obj_list_item.parameters;
       let parameter_call_input = "";
+      let transform_type =
```

Comment:
Here, transform_type will be a bool value. It just doesn't fit with the word since the word implies a string value. Recommend something like `is_tarnsform_type`